Remotion/Mixins/Core/Definitions/Building/AttributeDefinitionBuilder.cs

```diff
@@ -60,7 +61,7 @@ private bool IsIgnoredAttributeType (Type type)
     {
       Assertion.IsNotNull (type.Namespace);
       return type == typeof (SerializableAttribute)
-          || (typeof (ExtendsAttribute).Assembly.Equals (type.Assembly) && type.Namespace.StartsWith ("Remotion.Mixins"));
+          || (typeof (ExtendsAttribute).Assembly.Equals (type.Assembly) && type.GetNamespaceSafe().StartsWith ("Remotion.Mixins"));
```

Comment:
```suggestion
          || (typeof (ExtendsAttribute).Assembly.Equals (type.Assembly) && type.GetNamespaceChecked().StartsWith ("Remotion.Mixins"));
```

Given there's already a check for type.Namespace not being null.